utils/_context/_scenarios/parametric.py

```diff
@@ -444,39 +444,36 @@ def dotnet_library_factory():
         container_name="dotnet-test-api",
         container_tag="dotnet8_0-test-api",
         container_img=f"""
-FROM mcr.microsoft.com/dotnet/sdk:8.0 as build
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build
+WORKDIR /app
 
 # `binutils` is required by 'install_ddtrace.sh' to call 'strings' command
-RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y curl binutils
-WORKDIR /app
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y binutils
 
-# ensure that the Datadog.Trace.dlls are installed from /binaries
-COPY utils/build/docker/dotnet/install_ddtrace.sh utils/build/docker/dotnet/query-versions.fsx binaries* /binaries/
+COPY utils/build/docker/dotnet/install_ddtrace.sh utils/build/docker/dotnet/query-versions.fsx binaries/ /binaries/
 RUN /binaries/install_ddtrace.sh
 
-# restore nuget packages
+# dotnet restore
 COPY {dotnet_reldir}/ApmTestApi.csproj {dotnet_reldir}/nuget.config ./
 RUN dotnet restore "./ApmTestApi.csproj"
 
-# build and publish
+# dotnet publish
 COPY {dotnet_reldir} ./
-RUN dotnet publish --no-restore --configuration Release --output out
+# RUN dotnet publish --no-restore -c Release -f net8.0 -o out
+RUN dotnet publish --no-restore -c Release -o out
```

Comment:
Why change `--configuration` and `--output` to `-c` and `-o`?

Reply:
Same answer as above, to stick to the weblog dockerfile 😅